[#292]add ansible role to install node_exporter

jhd2best opened the pull request:
add an ansible role to install node_exporter systemd service

LeoHChen commented on ansible/playbooks/roles/node_exporter/tasks/main.yml:
```diff
@@ -0,0 +1,54 @@
+
+- name: add node_exporter user
+  become: true
+  shell: useradd -rs /bin/false node_exporter
```
use ansible user module
https://docs.ansible.com/ansible/latest/modules/user_module.html

jhd2best replied:
sure

jhd2best commented:
**Test Result:**
PASS

**Test Log:**
ansible-playbook -i inventory/haodi.hosts playbooks/install-node-exporter.yml -e 'inventory=test user=hmy'

PLAY [test] ******************************************************************************************************************************

TASK [Gathering Facts] *******************************************************************************************************************
ok: [52.191.166.139]

TASK [node_exporter : gather OS specific variables] **************************************************************************************
ok: [52.191.166.139] => (item=/Volumes/Project/Harmony/experiment-deploy/ansible/playbooks/roles/node_exporter/vars/Azure-CentOS.yml)

TASK [node_exporter : download node_exporter package] ************************************************************************************
ok: [52.191.166.139]

TASK [node_exporter : tar node_exporter package] *****************************************************************************************
changed: [52.191.166.139]

TASK [node_exporter : install node_exporter] *********************************************************************************************
changed: [52.191.166.139]

TASK [node_exporter : add node_exporter user] ********************************************************************************************
changed: [52.191.166.139]

TASK [node_exporter : copy node_exporter service files] **********************************************************************************
ok: [52.191.166.139]

TASK [node_exporter : enable node_exporter service] **************************************************************************************
changed: [52.191.166.139]

TASK [node_exporter : clear trash] *******************************************************************************************************
changed: [52.191.166.139] => (item=/home/hmy/node_exporter-0.18.1.linux-amd64.tar.gz)
changed: [52.191.166.139] => (item=/home/hmy/node_exporter-0.18.1.linux-amd64)

PLAY RECAP *******************************************************************************************************************************
52.191.166.139             : ok=9    changed=5    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
